images/velero-plugin-for-csi/metadata.yaml

```
@@ -0,0 +1,11 @@
+name: velero-plugin-for-csi
+image: cgr.dev/chainguard/velero-plugin-for-csi
+logo: https://storage.googleapis.com/chainguard-academy/logos/velero-plugin-for-csi.svg
+endoflife: ""
+console_summary: ""
+short_description: Velero plugins for integrating with CSI snapshot API
+compatibility_notes: ""
+readme_file: README.md
+upstream_url: https://github.com/vmware-tanzu/velero-plugin-for-csi
+keywords: []
```

To help our docs team :) 
```suggestion
keywords:
  - application
  - kubernetes
```